[{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/comments/326616284","html_url":"https://github.com/elastic/elasticsearch/issues/26475#issuecomment-326616284","issue_url":"https://api.github.com/repos/elastic/elasticsearch/issues/26475","id":326616284,"node_id":"MDEyOklzc3VlQ29tbWVudDMyNjYxNjI4NA==","user":{"login":"jasontedor","id":4744941,"node_id":"MDQ6VXNlcjQ3NDQ5NDE=","avatar_url":"https://avatars3.githubusercontent.com/u/4744941?v=4","gravatar_id":"","url":"https://api.github.com/users/jasontedor","html_url":"https://github.com/jasontedor","followers_url":"https://api.github.com/users/jasontedor/followers","following_url":"https://api.github.com/users/jasontedor/following{/other_user}","gists_url":"https://api.github.com/users/jasontedor/gists{/gist_id}","starred_url":"https://api.github.com/users/jasontedor/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/jasontedor/subscriptions","organizations_url":"https://api.github.com/users/jasontedor/orgs","repos_url":"https://api.github.com/users/jasontedor/repos","events_url":"https://api.github.com/users/jasontedor/events{/privacy}","received_events_url":"https://api.github.com/users/jasontedor/received_events","type":"User","site_admin":false},"created_at":"2017-09-01T15:50:54Z","updated_at":"2017-09-01T15:50:54Z","author_association":"MEMBER","body":"Elastic provides a [forum](https://discuss.elastic.co) for asking general questions and instead prefers to use GitHub only for verified bug reports and feature requests. There's an active community there that should be able to help get an answer to your question. As such, I hope you don't mind that I close this.","performed_via_github_app":null},{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/comments/326618189","html_url":"https://github.com/elastic/elasticsearch/issues/26475#issuecomment-326618189","issue_url":"https://api.github.com/repos/elastic/elasticsearch/issues/26475","id":326618189,"node_id":"MDEyOklzc3VlQ29tbWVudDMyNjYxODE4OQ==","user":{"login":"zqc0512","id":25192261,"node_id":"MDQ6VXNlcjI1MTkyMjYx","avatar_url":"https://avatars1.githubusercontent.com/u/25192261?v=4","gravatar_id":"","url":"https://api.github.com/users/zqc0512","html_url":"https://github.com/zqc0512","followers_url":"https://api.github.com/users/zqc0512/followers","following_url":"https://api.github.com/users/zqc0512/following{/other_user}","gists_url":"https://api.github.com/users/zqc0512/gists{/gist_id}","starred_url":"https://api.github.com/users/zqc0512/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/zqc0512/subscriptions","organizations_url":"https://api.github.com/users/zqc0512/orgs","repos_url":"https://api.github.com/users/zqc0512/repos","events_url":"https://api.github.com/users/zqc0512/events{/privacy}","received_events_url":"https://api.github.com/users/zqc0512/received_events","type":"User","site_admin":false},"created_at":"2017-09-01T15:58:35Z","updated_at":"2017-09-01T15:58:35Z","author_association":"NONE","body":"i think it a bug, a large env(20T + data 50000+shards) output the error,in other env (100G+ data) not output the error.\r\nsorry\r\n@jasontedor \r\n","performed_via_github_app":null},{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/comments/326712890","html_url":"https://github.com/elastic/elasticsearch/issues/26475#issuecomment-326712890","issue_url":"https://api.github.com/repos/elastic/elasticsearch/issues/26475","id":326712890,"node_id":"MDEyOklzc3VlQ29tbWVudDMyNjcxMjg5MA==","user":{"login":"jasontedor","id":4744941,"node_id":"MDQ6VXNlcjQ3NDQ5NDE=","avatar_url":"https://avatars3.githubusercontent.com/u/4744941?v=4","gravatar_id":"","url":"https://api.github.com/users/jasontedor","html_url":"https://github.com/jasontedor","followers_url":"https://api.github.com/users/jasontedor/followers","following_url":"https://api.github.com/users/jasontedor/following{/other_user}","gists_url":"https://api.github.com/users/jasontedor/gists{/gist_id}","starred_url":"https://api.github.com/users/jasontedor/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/jasontedor/subscriptions","organizations_url":"https://api.github.com/users/jasontedor/orgs","repos_url":"https://api.github.com/users/jasontedor/repos","events_url":"https://api.github.com/users/jasontedor/events{/privacy}","received_events_url":"https://api.github.com/users/jasontedor/received_events","type":"User","site_admin":false},"created_at":"2017-09-02T01:20:12Z","updated_at":"2017-09-02T01:20:12Z","author_association":"MEMBER","body":"It is possible that you're running into a related issue: #26293. We do not know the cause of this issue yet, all I can tell you is that we are taking it seriously.","performed_via_github_app":null}]